src/include/iscsi-lio-server/UI_dialogs.rb

```diff
@@ -436,8 +436,30 @@ def contents
     end
 
     def handle(event)
-      puts event
+      cmd = "systemctl"
       @service_status.handle_input(event["ID"])
+      service_enabled = @service_status.enabled_flag?
```

Comment:
Why don't you simply use `@service.enable` and `@service.disable` here instead of crafting the whole command manually?

This should work:
```ruby
if @service_status.enabled_flag?
  @service.enable
else
  @service.disable
end
```

And another option, if you prefer:
```ruby
if @service_status.enabled_flag?
  Service.enable("targetcli.service")
else
  Service.disable("targetcli.service")
end
```

That being said, it's kind of unexpected for a YaST module to change the configuration of the system right away on every user click, instead of waiting for the user to close "ok"/"finish" and make all the changes at once. But maybe `yast-isci-lio-server` doesn't follow that YaST convention in general.